### ISSUE TITLE
Ios support plan? 

### ISSUE THREAD

**Issue by kizovinh** [pulsejet/memories#998]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by pulsejet** [comment]
No plan at the moment. Happy to provide an overview of the NativeX API if someone is willing to contribute this.

**Comment by wonx** [comment]
You can just create a desktop shortcut from Safari and it works quite well.

**Comment by kizovinh** [comment]
> You can just create a desktop shortcut from Safari and it works quite well.

Yes I am using that solution, but native support is always better

**Comment by m-oehme** [comment]
Hey @pulsejet i started working on iOS client a couple months ago. I haven’t continued since then (work stuff)
Feel free to check it out: https://github.com/m-oehme/memories/tree/feature/ios_native_app

It’s more of an MLP for now. But would love if you could provide some short description of NativeX. Some apis were not clear to me Afair

**Comment by pulsejet** [comment]
Nice! I'm going to try putting together documentation for NativeX (https://github.com/pulsejet/memories/issues/1147). It's somewhat messy because there's no easy way to efficiently do communication between the webview and the native parts. If you have any specific questions in the meanwhile, send me a message on the discord dev channel.

**Comment by veenstrajelmer** [comment]
This would be an amazing additon. I am not using iOS myself, but know several people who do, and it would help with the adoption of nextcloud/memories as a photo solution. On android and in the browser it already works really amazing. A shortcut on the iOS desktop works, but this with this method local phone pictures cannot be included in the timeline.

**Comment by UseroftheGalaxy** [comment]
An iOS app that provides access to the photo management solution would benefit Nextcloud immensely. I can only speak for my family and friends, but nearly all photo management—albums, sharing, etc.—is done via smartphone. As long as there is no app that supports features like albums or sharing, they won't accept Nextcloud as their primary platform.

**Comment by vtmocanu** [comment]
Dont want to be just another +1, but yeah, I've been watching this issue for sometime in the hopes that this will get implemented on iOS. Really hope to see it happen someday.

**Comment by schorschie** [comment]
Hey, so the absence of that feature is currently a show stopper for me. I have my whole family on iPhones and we use the Apple Photos App and the shared foto albums in there as the core platform for sharing photos. I really want to move away from US clouds to a self hosted solution, but without an ios app my familiy will not join.

I'm ok with C#, but I don't even have a Mac to develop an iOS app. Is there any way I could contribute?

regards
Gregory

PS It's not even the AI features in Memories I need, but a simple shared photo album would be sufficent.

**Comment by pjator** [comment]
Well sadly I do not have skills enough to contribute but have been a long time user of Nextcloud but realizing more and more that I am more a infrastructure maintainer now then a user. 
I would love to see a separation of the current Nextcloud app and just see that as a file app the same way Talk is a separate app. And of course the natural extension that Memories (due to its superiour performance compared to photos) becomes a seperat app and function withing nextcloud. 
This would be befenital both for Homelabbers and corporate users that you the indivial functions where more independent so you can adjust your infrastructure and usecase accordingly. 

To be honest even if I think I have a great setup with an FPM/NGINX setup on S3 Object Storage with collabra and high performance talk backend setup I am still considering migration to a Owncloud/Immich/Matrix stack that seperates the services due to the fact that I to often find myself fixing one thing in Nextcloud just to brake something I did not think of due to the cross dependenzies. 
But i guess this post is just "screaming in the wind" and I have a lot of respect that it is difficult to value all wishes on and opensource project.

**Comment by getwet** [comment]
A nativ app would be my kick to put all my photos to memories.
The web app is great, but I don´t like PWA´s so much and I´m using ios 90% per day.
